pkg/controller.v1/common/job_controller.go

```diff
@@ -26,8 +26,8 @@ import (
 	"k8s.io/client-go/tools/cache"
 	"k8s.io/client-go/tools/record"
 	"k8s.io/client-go/util/workqueue"
-	"volcano.sh/volcano/pkg/apis/scheduling/v1beta1"
-	volcanoclient "volcano.sh/volcano/pkg/client/clientset/versioned"
+	"volcano.sh/apis/pkg/apis/scheduling/v1beta1"
+	volcanoclient "volcano.sh/apis/pkg/client/clientset/versioned"
```

Comment:
There is another place using volcano.sh/volcano directly.

link: [https://github.com/kubeflow/common/blob/master/pkg/controller.v1/common/job.go#L18](https://github.com/kubeflow/common/blob/master/pkg/controller.v1/common/job.go#L18)